scripts/cell_simulation.py

```diff
@@ -67,7 +94,8 @@ def create_parser() -> argparse.Namespace:
         "--observe_homozygous",
         required=True,
         help="Observing homozygous mutations",
-        type=bool,
+        choices=[False, True],
+        type=t_or_f,
```

Comment:
There was a bug here. `cell_simulation.py` did not parse a string "False" correctly. It always parsed `True`. Hence, this custom function `t_or_f`. Please let me know if there is a more elegant way.

Reply:
Actually, I'd recommend not making it required and setting it to `False` by default.

This should work:
```
parser.add_argument(
    "--observe-homozygous",
    help="Observing homozygous mutations. By default False.",
    action="store_true",
)
```